ecosystem/sep-0010.md

```diff
@@ -22,18 +22,23 @@ The authentication flow is as follows:
 
 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
-1. The client signs the transaction using the secret key for the user's Stellar account
+1. The client signs the transaction using the secret key(s) of signers of the user's Stellar account
 1. The client submits the signed challenge back to the server using [`token`](#token) endpoint
-1. If the signature checks out, the server responds with a [JWT](https://jwt.io) that represents the user's session
+1. The server gets the signers of the user's account
+1. The server verifies the signatures
+1. The server verifies the weight provided by the signers meets the required threshold(s)
+1. The server responds with a [JWT](https://jwt.io) that represents the user's session
 1. Any future calls to the server can be authenticated by including the JWT as a parameter
 
 The flow achieves several things:
 
 * Both client and server part can be implemented using well-established Stellar libraries
-* The client can verify that the server holds the secret key to a particular account
-* The server can verify that the client holds the secret key to their account
-* The client is able to prove their identity using a Ledger or other hardware wallet as well as by having direct access to the secret key
-* The server can chose its own timeout for the user's session
+* The client can verify that the server holds the secret key to a particular public key
+* The server can verify that the client holds the secret key(s) of signers of the user's Stellar account
+* The client is able to prove their identity using a Ledger or other hardware wallet as well as by having direct access to the secret key(s)
+* The server can choose its own timeout for the user's session
+* The server can choose the required threshold(s) a user needs on the account
+* Other servers can choose the required threshold(s) a user needs on the account for specific functionality
```

Comment:
It may be a little unclear what "Other servers" refers to. I think it's other servers (probably controlled by the same entity as the main server) that also use the JWT to decide whether the user is authorized for their specific functionality.

Reply:
I've attempted to provide more detail to who those servers are in 4f06a28 without spending too many words on it.